packages/node/example/bootstrappedApp/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "@inrupt/demo-app-authn-node",
+  "version": "1.0.0",
+  "description": "Demo client application for authentication in NodeJS",
+  "main": "src/index.js",
+  "scripts": {
+    "lint": "eslint \\\"src/**/*.js\\\"",
+    "start": "./node_modules/.bin/nodemon src/index.js",
+    "dev": "npm run start",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/inrupt/solid-client-authn.git"
```

Comment:
Although this works (via a redirect it seems), it's missing `-js` suffix, i.e. shouldn't it be `git+https://github.com/inrupt/solid-client-authn-js.git`

Reply:
Fixed in 81552d136cbe99d4bf1e305c04105e1568ff9571